pmc/journal_recommendation/journal_recommendation_figure_generator.py

```diff
@@ -0,0 +1,570 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Journal Recommendation - Figure Generator
+
+# In[1]:
+
+
+from collections import Counter
+import itertools
+import os
+from pathlib import Path
+import pickle
+import random
+
+import numpy as np
+import pandas as pd
+from scipy.spatial.distance import cdist
+from sklearn.decomposition import PCA
+from tqdm import tqdm
+
+from saucie_modules import SAUCIE, Loader
+import tensorflow.compat.v1 as tf
+
+import plotnine as p9
+
+
+# In[2]:
+
+
+# Set up porting from python to R
+# and R to python :mindblown:
+
+import rpy2.rinterface
+get_ipython().run_line_magic('load_ext', 'rpy2.ipython')
+
+
+# # Plot Accuracy Results
+
+# In[3]:
+
+
+results = [
+    {"value":.00517, "model":"random_baseline", "distance":"N/A", "dataset":"train"},
+    {"value":.39982, "model":"paper_paper", "distance":"euclidean", "dataset":"train"},
+    {"value":.37340, "model":"centroid", "distance":"euclidean", "dataset":"train"},
+    {"value":.39824, "model":"paper_paper", "distance":"manhattan", "dataset":"train"},
+    {"value":.39824, "model":"centroid", "distance":"manhattan", "dataset":"train"},
+    {"value":.00445, "model":"random_baseline", "distance":"N/A", "dataset":"test"},
+    {"value":.20638, "model":"paper_paper", "distance":"euclidean", "dataset":"test"},
+    {"value":.21636, "model":"centroid", "distance":"euclidean", "dataset":"test"},
+    {"value":.20380, "model":"paper_paper", "distance":"manhattan", "dataset":"test"},
+    {"value":.21523, "model":"centroid", "distance":"manhattan", "dataset":"test"}
+]
```

Comment:
These results come from the journal_recommendation_expriment notebook. It was easier to hard code the results than rerun that notebook as it can take up to days to run. This situation it makes sense to just let it be, but good to note for the future.